apps.yaml

```diff
@@ -262,4 +262,40 @@ apps:
     tls:
       key_vault_name: "acmedtscftsboxintsvc"
       name: "wildcard-hmcts-net"
-    internalUrl: "https://backstage-sandbox.hmcts.net/"
\ No newline at end of file
+    internalUrl: "https://backstage-sandbox.hmcts.net/"
+  - name: "Panorama - Sandbox"
+    externalUrl: "https://panorama-sandbox.hmcts.net"
+    internalUrl: "https://panorama-sandbox.hmcts.net"
+    logoUrl: https://raw.githubusercontent.com/hmcts/azure-app-proxy/main/logos/paloalto.png
+    userAssignmentRequired: false
+    appRoleAssignments:
+      - DTS Platform Operations
+      - dcd_team_secops_v2
+      - DTS Firewall Readers
+    graphApiPermissions:
+      - "User.Read.All"
+      - "Group.Read.All"
+      - "People.Read.All"
```

Comment:
I don't think these are needed, app reg currently doesn't seem to have any set up?